config/dev1/sys.config

```diff
@@ -11,7 +11,14 @@
           {acceptors, 10},
           {port, 3113},
           {websocket, [ {port, 3114},
-                        {tasks, 200}
+                        {tasks, 200},
+                        {handlers, 100}
+                      ]}
+          ]},
+      {channel, [
+          {websocket, [ {port, 3014},
```

Comment:
Did you run Python UAT tests? I expect they shall work though checking.

Reply:
Yes, I've ran python test locally and they've passed